doc/manual.rst

```diff
@@ -4493,8 +4515,10 @@ Open Interest Structure
 
    {
        symbol: 'BTC/USDT',
-       baseVolume: 80872.801,
-       quoteVolume: 3508262107.38,
+       baseVolume: 80872.801,  // deprecated
+       quoteVolume: 3508262107.38, // deprecated
```

Comment:
If you're going to deprecate these fields, I think you should create another PR where the new fields are added for every exchange that has `parseOpenInterest`